packages/fast-components-react-base/src/media/Media.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import Foundation, { HandledProps } from "../foundation";
+import {IMediaProps, MediaHTMLTags} from "./Media.props";
+import {IMediaClassNameContract, IManagedClasses} from "@microsoft/fast-components-class-name-contracts";
+
+/* tslint:disable-next-line */
+class Media extends Foundation<IMediaProps & IManagedClasses<IMediaClassNameContract>,  React.HTMLAttributes<HTMLImageElement | HTMLPictureElement>, {}> {
+    protected handledProps: HandledProps<IMediaProps & IManagedClasses<IMediaClassNameContract>> = {
+        managedClasses: void 0,
+        alt: void 0
+    };
+
+    generatePicture(): React.ReactElement<HTMLPictureElement> {
+         let spaceSplit = /\s* /;
+         let commaSplit = /\s*, \s*/;
+         let input = this.props.pictureSrcSet.split(commaSplit);
+         let first = input[0].split(spaceSplit);
+         let sources = input.map((item , index) => {
+            let output = item.split(spaceSplit);
+            return (
+                <source key={index} srcSet={output[0]} media={`(min-width: ${output[1]})`} />
+            );
+         });
+
+         return (
+            <picture
+                className={this.generateClassNames()}>
+                {sources}
+                <img srcSet={first[0]} src={first[0]} alt={this.props.alt} />
```

Comment:
what happens here if the string is not in sequential order?

Reply:
It has to be just like (Correct me if I'm wrong) srcSet on and <img /> needs to be in sequential order to function

Reply:
http://alistapart.com/article/responsive-images-in-practice